src/canvas.ts

```diff
@@ -60,14 +78,27 @@ export const createCanvas = (id: string) => {
       window.origin,
     );
   });
-  window.addEventListener('resize', () => resizeCanvas(canvas));
+
+  window.addEventListener('resize', () => resizeCanvas(canvas, { width, height }));
```

Comment:
```suggestion
 if (!width && !height) window.addEventListener('resize', () => resizeCanvas(canvas));
```
이러면 의미없는 핸들러 호출이 상당히 많이 발생할거라, resizeCanvas 내부를 수정하기 보다는 여기에서 이런 식으로 가는게 맞다고 봅니다.

Reply:
+) 여기에 더해, 이전 PR에서 제가 놓친 부분인데, 그냥 이벤트 리스너는 하나만 걸고, 하나의 핸들러 내에서 `canvasMap` 내에 접근해 처리를 하는 방향이 좋을 것 같아요. 왜냐하면 기존에 작성된 `setViewPort`이라는 함수도 굳이 여러번 실행될 것을 의도하고 작성한 건 아니기 때문입니다.